x/auth/simulation/genesis.go

```diff
@@ -26,13 +26,6 @@ func RandomGenesisAccounts(simState *module.SimulationState) types.GenesisAccoun
 	for i, acc := range simState.Accounts {
 		bacc := types.NewBaseAccountWithAddress(acc.Address)
 
-		// check if vesting module is enabled
```

Comment:
Could you revert this? This was a regression fix: https://github.com/cosmos/cosmos-sdk/pull/22721. We cannot expect the vesting module to be present in new chains.

Reply:
good eye, thanks!